api/envoy/config/core/v3/address.proto

```diff
@@ -32,15 +32,19 @@ message Pipe {
 }
 
 // The address represents an envoy internal listener.
-// [#comment: TODO(lambdai): Make this address available for listener and endpoint.
 // TODO(asraa): When address available, remove workaround from test/server/server_fuzz_test.cc:30.]
 message EnvoyInternalAddress {
```

Comment:
Maybe worth to note that this is only used for upstream endpoint address.

Reply:
Added.